RegistryMetadata.md

```diff
@@ -0,0 +1,25 @@
+# Registry Metadata
+
+## Introduction
+
+We have 3 data exchange mechanisms as described in the [Architecture Principles](README.md).  In as much as possible, we want to create the same metadata for each method, and expose that metadata through a common query device: The SmartAPI registry.   Below, these methods are abbreviated as S (for SmartAPI), T (for Translator Reasoner API), and G (for Knowledge Graph Exchange).  
+
+This document describes, broadly, the particular metadata that Translator data exchange methods must or should expose. The technical details of how this metadata is specified is located [here]() for SmartAPI-type interfaces, [here]() for Translator Reasoner APIs, and [here]() for Knowledge Graph Exchange files.  
+
+## Change Management
+
+Changes to this document must be made via pull requests.   Questions or discussion around a topic that is not easily related to a specific pull request occurs in github issues.
+
+## Metadata
+
+1. Access information (STG) 
+1. Node types (STG)
+1. Predicates (STG)
+1. Identifier schemes (STG)
+1. Evidence / Provenance (STG)
+1. Node Properties (STG)
+1. Edge Properties (STG)
```

Comment:
It seems unrealistic to ask ARAs to describe their support for types of evidence/provenance and node and edge properties. Maybe predicates and node types, too. These will depend on the capabilities of the underlying KPs, which may be evolving. If a service depends on other registered services for these things, could it just point to them?

Reply:
Hi @patrickkwang, I agree that we have to be careful not to go overboard here, but I also wonder if @mbrush's directed dialog with the community about E/P/C data models will point to some very basic minimal set of metadata describing the general nature of the available E/P/C metadata for a given Translator knowledge graph source.   Perhaps some of the ARS and ARA use cases for E/P/C will point to the helpful scope and utility of such E/P/C metadata.

Reply:
> It seems unrealistic to ask ARAs to describe their support for types of evidence/provenance and node and edge properties. Maybe predicates and node types, too. These will depend on the capabilities of the underlying KPs, which may be evolving. If a service depends on other registered services for these things, could it just point to them?

This may have been the reason why ARAs are not supposed to do any ETL. I was under the impression that ARAs that store any KGs would do it as a KP.

Reply:
Agreed.   For some operations (fill, overlay) then we might want that metadata attached to a trapi interface, but perhaps for others (whatever the ara operation is) it might not make sense... Not 100% clear how to make that distinction here.  It could be ARA/KP, or it could be at a TRAPI operation level.